### PR TITLE
Remove newline to show FB and Twitter buttons on same line #4821

### LINKED ISSUE #4821: Remove newline to show FB and Twitter buttons on same line
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update**: 
Remove newline between the FB and Twitter buttons 
share_buttons

Let's shift them on the same line by updating file. Also, there's a typo in the comment (at line 42) in the file, please remove that too. https://github.com/publiclab/plots2/blob/master/app/views/sidebar/_dashboard.html.erb#L42-L47

Also, attach the screenshot of the updated view. 

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by swiatek7**
Fixes #4821

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] PR is descriptively titled 📑
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

~~Removing \<br> tag was not simply enough, since facebook button root node is a div with display block.
I am not that happy with this solution, because it relies on how Facebook renders its button, which can change at any time.
Other solution I came up with was adding float:left and margin-right to twitter button, then it does not matter that Facebook button is block element, but still it relies on how twitter's script renders its button in iframe.
Last solution I came up, but could not make work was wrapping both of them in divs and then position them based on our containers div with no regard what is inside.~~

I have managed to implement this feature with wrapping both buttons in divs. Not it **is** independent from the way twitter or facebook renders its buttons.

remove newline to show fb and twitter buttons on same line 4821

remove newline to show fb and twitter buttons on same line 4821

### CODE REVIEW

**Comment by swiatek7** [comment]
Hi  @publiclab/reviewers
I kindly ask your for the review :)

**Comment by swiatek7** [comment]
* @publiclab/reviewers I have pushed new code.

File: app/views/sidebar/_dashboard.html.erb [hunk 1]
```diff
@@ -39,10 +39,13 @@
...
-    <br>
+    <!-- Twitter follow button, options can be adjusted here: https://publish.twitter.com/# -->
+    <div class="social-media-btn">
+      <a href="https://twitter.com/PublicLab?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-size="large" data-show-count="false">Follow @PublicLab</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+    </div>
     <!-- Facebook like button, options can be adjusted here: https://developers.facebook.com/docs/plugins/like-button/  -->
-    <div class="fb-like" data-href="https://www.facebook.com/PublicLab/" data-layout="button_count" data-action="like" data-size="large" data-show-faces="false" data-share="false"></div>
+    <div class="social-media-btn">  
+      <div class="fb-like" data-href="https://www.facebook.com/PublicLab/" data-layout="button_count" data-action="like" data-size="large" data-show-faces="false" data-share="false"></div>
+    </div
```
Comment (gauravano) [inline app/views/sidebar/_dashboard.html.erb]:
```suggestion
    </div>
```

**Comment by meet2410shah** [comment]
I want to claim this.

**Comment by swiatek7** [comment]
@gauravano 

For some reason it does not work to add this gif in description of PR.

remove newline to show fb and twitter buttons on same line 4821

**Comment by swiatek7** [comment]
> Could you please provide a GIF showing the status of buttons on different screen widths? Thanks!

Done :)

**Comment by gauravano** [comment]
Merged :100: :+1:!

Hey @swiatek7, thanks for nice work. We would love to have more help from you. If you are interested in solving more issues then check here - https://code.publiclab.org/#r=all for a new issue. In case, you are unable to find an issue, feel free to comment here. Thanks!

**Comment by gauravano** [comment]
Hey @meet2410shah, please check other issues as this one is complete. Thanks!

**Comment by swiatek7** [comment]
Thanks @gauravano :) 
I will definitely check out more issues and I will try to contribute.

**Pull request merged**